internal/masterservice/master_service.go

```diff
@@ -52,6 +52,16 @@ import (
 
 // ------------------ struct -----------------------
 
+// DdOperation used to save ddMsg into ETCD
+type DdOperation struct {
+	Body         []byte
+	Body1        []byte // used for DdCreateCollection only
+	Type         string
+	CollectionID typeutil.UniqueID
+	PartitionID  typeutil.UniqueID
+	Send         bool
+}
```

Comment:
please add annotation for this struct, and field of `Body` and `Body1` should be string